src/ucp/rndv/proto_rndv.inl

```diff
@@ -386,4 +386,15 @@ ucp_proto_rndv_recv_complete(ucp_request_t *req)
     return ucp_proto_rndv_recv_complete_status(req, rreq->status);
 }
 
+static UCS_F_ALWAYS_INLINE void
+ucp_proto_rndv_ppln_abort(ucp_request_t *req, ucs_status_t status)
```

Comment:
minor: this function is supposed to be called by pointer on non-fast path, no need inline and static, i.e. it can be declared in some common `.h` and defined in `.c`

Reply:
Done